bears/general/KeywordBear.py

```diff
@@ -36,4 +94,5 @@ def run(self, filename, file, keywords: list):
                     column=keyword.start() + 1,
                     end_line=line_number + 1,
                     end_column=keyword.end() + 1,
-                    severity=RESULT_SEVERITY.INFO)
+                    severity=RESULT_SEVERITY.INFO,
+                    diffs=diffs)
```

Comment:
as everywhere `line_number + 1` is used, you maybe can modify `enumerate` above:
```python
for line_number, line in enumerate(file, start=1)
```
then you can replace every occurrence of `line_number + 1` with `line_number`.

Maybe you can make a new commit before this one which changes the `enumerate` for the old version :+1: